Dockerfile.docs

```diff
@@ -1,19 +1,21 @@
 FROM python:3.8-slim
 
+# build-essential is required to build a wheel for ciso8601
+RUN apt update && apt install -y build-essential
+
 RUN python -m pip install --upgrade pip
 RUN python -m pip install mkdocs mkdocs-material pdocs
 
 COPY . /opt/src
 
 WORKDIR /opt/src
 
-RUN python -m pip install -e \
+RUN python -m pip install \
```

Comment:
`-e` isn't needed in this case, and it only affects the first package name after it (rather than all of them)